test/debug.jl

```diff
@@ -9,6 +9,18 @@ function step_through(frame)
     return get_return(r)
 end
 
```

Comment:
I wonder if we should have a function that tries to step through a function with all different commands we have. And one that just does a random command each time :D

Reply:
#157